app/assets/stylesheets/components/_border.scss

```diff
@@ -10,6 +10,10 @@
 .rounded-xl { border-radius: $border-radius-xl; }
 .rounded-xxl { border-radius: $border-radius-xxl; }
 
+.border-blue-medium {
```

Comment:
Our default border color is nearly identical to this shade of blue so for consistency's sake I think we should just stick with what we have.